src/Mvc/Mvc.ViewFeatures/src/RazorComponents/Prerendering/IComponentPrerenderer.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components.Server
+{
+    /// <summary>
+    /// Prerrenders <see cref="IComponent"/> instances.
+    /// </summary>
+    public interface IComponentPrerenderer
```

Comment:
Is there a follow up issue to move this to Components.Server? This looks like the wrong place for this to define this

Reply:
There are plans

Reply:
We are debating how to best do this. In essence there are two possibilities. We drop it into
Microsoft.AspNetCore.Components
we create a separate dll
Microsoft.AspNetCore.Components.Http/Core/Abstractions or similar

For preview3 it was agreed that this would remain here as comonents.server referenced MVC transitively up until very recently.

Reply:
Could you file an issue for this please?